src/NuGet.Clients/NuGet.Tools/Properties/AssemblyInfo.cs

```diff
@@ -13,6 +13,7 @@
 [assembly: ProvideCodeBase(CodeBase = @"$PackageFolder$\Lucene.Net.dll")]
 [assembly: ProvideCodeBase(CodeBase = @"$PackageFolder$\Microsoft.Web.XmlTransform.dll")]
 [assembly: ProvideCodeBase(CodeBase = @"$PackageFolder$\Newtonsoft.Json.dll")]
+[assembly: ProvideCodeBase(CodeBase = @"$PackageFolder$\NugetRecommender.VisualStudio.Contracts.dll")]
```

Comment:
Please change spelling to: NuGet

Reply:
Removed here, but fixed spelling in other locations.